### ISSUE TITLE
STEAMPUNK: Incompatible syntax

### ISSUE THREAD

**Issue by sapbtp-bmsudhakaran** [sbcgua/abap_mustache#14]
Branch: Steampunk
Class: zcl_mustache_render
File: abap_mustache/src/zcl_mustache_render.clas.abap
Line: 114
Issue: Describe is not allowed in the current STEAMPUNK version

**Comment by sbcgua** [comment]
Indeed. Thanks for pointing out. I'll try to have a look when I have time, but a PR from your side is always welcomed :)
